test/functional/test_framework/address.py

```diff
@@ -159,6 +162,14 @@ def check_script(script):
     assert False
 
 
+def bech32_to_bytes(address):
+    hrp = address[:2]
+    version, payload = decode_segwit_address(hrp,address)
+    if version is None:
+        return None, None
```

Comment:
there should be a space between function arguments and I'd return a tuple here to stay consistent with the other functions in the `test_framework/segwit_addr.py` file:

```suggestion
    version, payload = decode_segwit_address(hrp, address)
    if version is None:
        return (None, None)
```